package.json

```diff
@@ -28,6 +28,7 @@
     "cy:open": "cypress open",
     "build-and-test": "npm run build -s && npm run test:build -s",
     "docs:build": "docz build",
+    "postdocs:build": "cpy \"**/*\" ../../.docz/dist --cwd=\"other/public\" --parents",
```

Comment:
Because rawgit is shutting down, we need to find another place to host static files. So I created a `other/public` directory. All the files in there will be copied as-is to the `.docz/dist` directory which is what netlify is configured to publish.